theatre/studio/src/panels/DetailPanel/propEditors/utils/useEditingToolsForPrimitivePropInDetailsPanel.tsx

```diff
@@ -96,6 +103,7 @@ export function useEditingToolsForPrimitiveProp<
       [],
     )
 
+    // hmmm... where was this going?
```

Comment:
lol, delete comment?

Reply:
did not resolve: This comment remains. I'm fine with the spirit of this comment, but it is ambiguous what it means. Is it a question of what the commented out code below it is doing? Why is that code commented out? Should it just be removed?